api/src/main/java/javax/observer/collection/CollectionObservable.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package javax.observer.collection;
+
+import java.util.Collection;
+import javax.observer.Subscription;
+
+/**
+ * 
+ * @param <T> element type of the collection
+ * @param <OC> observed collection type
+ * @param <CC> collection change type
+ * 
+ * @author Michael Hoffer (info@michaelhoffer.de)
+ */
+public interface CollectionObservable<T, OC extends Collection<T>, CC extends CollectionChange<T>> {
```

Comment:
Ok, I understand the use case and in this scenario the name makes sense. At the end most people will simple add listener to the ObservableCollection but for more specific use cases you can create a CollectionObservable that observers several collections. This sounds like a good solution for me.